requirements.txt

```diff
@@ -9,6 +9,7 @@ Flask>=1.0.2 # BSD
 iso8601>=0.1.11 # MIT
 keystoneauth1>=3.13.0 # Apache-2.0
 keystonemiddleware>=4.17.0 # Apache-2.0
+kubernetes>=10.0.1 # Apache-2.0
```

Comment:
Just make sure this is in-line with the openstack `requirements` repo for Train

Reply:
This should be fine, it matches the upper-constraint for train https://github.com/openstack/requirements/blob/stable/train/upper-constraints.txt#L488